cdk/resources/HealthCheckMqttBridge.ts

```diff
@@ -17,20 +18,28 @@ import { Scope } from '../../util/settings.js'
 
 export type BridgeImageSettings = BridgeSettings
 
+export type CoAPSimulatorImage = {
+	imageTag: string
+	repositoryName: string
+}
+
```

Comment:
CoAP health check should be a separate resource (because this is `HealthCheckMqttBridge`)